### PR TITLE
add updates

### PR DESCRIPTION
**Pull request by kkong87**
Add logic for autamated updates

### CODE REVIEW

**Comment by nandadubey** [comment]
can you add Jira and some details please

File: woocommerce-gateway-pb.php [hunk 1]
```diff
@@ -45,7 +46,7 @@
 }
 
 /**
- * Returns Affirm.
+ * Returns Paybrigt.
```
[the remark this answers is not in the document]
Comment (kkong87) [reply woocommerce-gateway-pb.php:49]: Yes, this is the paybright extension

File: includes/class-wc-gateway-paybright.php [hunk 2]
```diff
@@ -357,11 +355,11 @@ public function process_refund( $order_id, $amount = null, $reason = null ) {
...
 			)
 		);
-		if ( is_wp_error( $response['body'] ) ) {
-			$error_message = $response['body']->get_error_message();
-			echo "Something went wrong: '" . esc_attr( $error_message ) . "'" . esc_html( $response['body']->get_error_message() );
-		} else {
 
+		if ( is_wp_error( $response ) ) {
+			$error_message = $response->get_error_message();
+			echo "Something went wrong: '" . esc_attr( $error_message ) . "'" . esc_html( $response->get_error_message() );
```
[the remark this answers is not in the document]
Comment (kkong87) [reply includes/class-wc-gateway-paybright.php:361]: This echo gets show on the frontend

File: woocommerce-gateway-pb.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (taehyunlim) [reply woocommerce-gateway-pb.php:49]: missing an `h`

**Review by taehyunlim — APPROVED** [review]
lgtm

**Pull request merged**